This patch could be easily merged with the FreeBSD one, but we don't
because the files have no CVS header and then auto syncing overwrites
the local difference we make.  <grobian@gentoo.org>

--- gencat/genlib.c
+++ gencat/genlib.c
@@ -54,7 +54,7 @@
 #include <unistd.h>
 #endif
 
-#if !defined(__linux__) && !defined(__FreeBSD__)
+#if !defined(__linux__) && !defined(__FreeBSD__) && !defined(__APPLE__)
 #include <memory.h>
 static int bcopy(src, dst, length)
 char *src, *dst;
